config/development.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
again, don't see the paint to use JS for config files. At some point someone's going to add JS code there...

Reply:
ok, require works with json so I will change.